Reputation Oracle - Store operator information on chain (#1599)
humanprotocol/human-protocol: docs/sdk/changelog.md

# Changelog

### Added

> - **Added new operator module:** new module including old methods and a new one to return operators of a reputation network.

Reviewer: ```suggestion
- **Added OperatorUtils module:** new module for fetching leaders and operators information.
```

### Changed

- **Update ethers version:** update ethers to version 6.
- **Update cancel escrow method:** return transaction hash and refunded amount.

### Deprecated

### Removed

> - **Remove staking utils module:** moved to a new module with a more descriptive name

Reviewer: ```suggestion
- **Remove staking utils module:** the methods from this module have been moved  into the new OperatorUtils module.
```
